tools/proofers/spellcheck.inc

```diff
@@ -206,6 +207,7 @@ if ($user->profile->i_type == 1) {
             id="spsaveandnext"
             value="<?php echo attr_safe(_("Save as 'Done' & Proofread Next Page")); ?>"
             title="<?php echo attr_safe(_("Save page as done and proofread the next available page")); ?>"
+            onClick="return submitJustOnce(this);",
```

Comment:
Errant trailing comma.

Reply:
Thanks - copied from different context in `button_defs.inc` and not adjusted properly